### ISSUE TITLE
totw Windcaller Donnel won't die

### ISSUE THREAD

**Issue by Splooder24** [project-rubika/issues#501]
**Describe the bug**

**To Reproduce**
4. See error or Notice this behavior

**Expected behavior**

**Screenshots**

**Character info:**

```
Copy/paste the output of /getfull here
```

**Playfield info:**

```
Copy/paste the output of Shift+F9 here
```
----

**Comment by prkdrake** [comment]
Seems to be another instance of #393

**Comment by Splooder24** [comment]
i will delete this post sorry

**Comment by prkdrake** [comment]
Closed as duplicate of #393
